### ISSUE TITLE
Which model did you choose to publish?

### ISSUE THREAD

**Issue by trisct** [cogaplex-bts/bts#93]
Hi. Thanks for this great work.

I have a quick question about your published model. In the training code I find that you save a model if it is currently the best for a certain evaluation metric. I wonder how you chose which one to publish. Did you choose one that is overall the best among all saved models to publish or did you choose the one saved at the last epoch?

**Comment by cogaplex-bts** [comment]
@trisct Sorry for this late reply. We choose the best model from the online evaluation within given number of epochs.
